app/src/main/java/org/bostwickenator/ftpuploader/SettingsStore.java

```diff
@@ -22,7 +22,7 @@
     private JSONObject settings;
     private final File settingsFile = FileGetter.getFile("SET.JSN");
 
-    public static SettingsStore getSettingsStore() {
+    public static SettingsStore settingsStore() {
```

Comment:
Not a fan of this naming refactor, it creates the potential to confuse you, suddenly you might be doing `new settingsStore()` not realizing this is a  monolithic pattern.

A more ideal name would be the previous or even `getSettingsStoreInstance()`.

Reply:
Using get for getters is somehow old baken. I find it much more readable. Think this is preferences. Instantiating it with new () is not possible as the constructor is private...